### ISSUE TITLE
Search: change "Where" to current folder (or Files) when there's no document opened

### ISSUE THREAD

**Issue by Defman21** [Komodo/KomodoEdit#2270]
### Short Summary

It works fine when I press ctrl+f if I have a selected file in Places and it works fine when I press ctrl+f in an opened file, but if I don't have any files opened and press ctrl+f and try to search something, I'll get this error: `unexpected error: TypeError: view.scintilla is undefined` (it's not anywhere in the log, just a notification).

### Platform Information

*Komodo Edit or IDE?* IDE
*Komodo Version?* 10.2 nightly
*Operating System (and version)?* elementary OS 0.4

### Additional Information

An user screenshot from the forums:
![image](https://cloud.githubusercontent.com/assets/7100645/22030333/5a442ede-dcee-11e6-8523-d09ec993095b.png)

**Comment by Naatan** [comment]
We'll need to show a proper error message (cannot search without a file open).

**Comment by Defman21** [comment]
Yeah of course, but I also want to search be smarty so if I press ctrl+f not in Places but in the main Komodo area and there are no files opened, it will search for files then. It feels much intuitive for me :)

**Comment by cgchoffman** [comment]
Was this fixed?  I can't reproduce this.  @Defman21, are you able to reproduce this still?

**Comment by Defman21** [comment]
Yes. Open the Find pane, close every file you have opened. Pick `Current Document` in the `Where` dropdown list.

**Comment by cgchoffman** [comment]
I thought the issue is that it defaults to `Current Document`?  Obviously it'll fail if you select current doc.

**Comment by Defman21** [comment]
The point is that the error message is very unfriendly to the user.

**Comment by cgchoffman** [comment]
Ahh ok.  The title is misleading.

**Comment by Defman21** [comment]
I think the new one is fine.
